__tests__/app.test.js

```diff
@@ -17,4 +18,41 @@ describe('', () => {
 })
     })
     
+});
+
+describe('test for /api endpoint', () => {
+    test('gets 200', async () => {
+    const {body}  = await request(app).get('/api').expect(200);
+    const expected = {
```

Comment:
We want this test to be dynamic - we dont want to have to keep going back to this test and adding in new endpoints every time we update our endpoints.json file and this test will break if we don't. instead we can directly require in our endpoints.json file here and test the output against that - the endpoints.json file will always be up to date.